src/MensaWidget/MWCanteen.class.st

```diff
@@ -0,0 +1,114 @@
+"
+A MWCanteen is a object that stores the data of a canteen for a certain date with all available meals.
+
+Instance Variables
+	date:		<String>
+	id:		<String>
+	meals:		<String>
+	name:		<OrderedCollection>
+
+date
+	- the data the meals are available
+
+id
+	- the id correlating to id at the api endpoint
+
+meals
+	- array of all available meals as meal objects
+
+name
+	- name of the canteen
+
+"
+Class {
+	#name : #MWCanteen,
+	#superclass : #Object,
+	#instVars : [
+		'id',
+		'name',
+		'date',
+		'meals'
+	],
+	#category : #MensaWidget,
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:24'
+}
+
+{
+	#category : #'Constructor ',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:16'
+}
+MWCanteen class >> newWithId: anId Name: aName Date: aDate Meals: aMealList [
+
+	| newInstance | 
+	
+	newInstance := self new.
+	newInstance id: anId;
+				name: aName;
```

Comment:
do we want to agree on making a double indent when cascading?

Reply:
done